src/core/drive/visit.ts

```diff
@@ -313,15 +314,19 @@ export class Visit implements FetchRequestDelegate {
   }
 
   followRedirect() {
-    if (this.redirectedToLocation && !this.followedRedirect && this.response?.redirected) {
-      this.adapter.visitProposedToLocation(this.redirectedToLocation, {
+    if (this.hasReplaceAction()) {
+      this.adapter.visitProposedToLocation(this.redirectedToLocation!, {
         action: "replace",
         response: this.response,
       })
       this.followedRedirect = true
     }
   }
 
+  hasReplaceAction() {
```

Comment:
If this were declared as a property, it could be invoked without as property access instead of a function call (that is, without the `()`):

```js
get willRedirectAsReplaceAction() {
  return this.redirectedToLocation && !this.followedRedirect && this.response?.redirected
}

// elsewhere
this.willRedirectAsReplaceAction
```

Reply:
Thank you for comment. I will change this lines.

Reply:
Done in 9ac687a5fa5ade3da95af806cb7d303caa9c5d1b. Thank you.